packages/web/src/pages/sign-up-page/components/MobileNavHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  Box,
+  Flex,
+  IconAudiusLogoHorizontal,
+  IconCaretLeft,
+  PlainButton,
+  iconSizes
+} from '@audius/harmony'
+import { useHistory } from 'react-router-dom'
+
+type MobileNavHeaderProps = {
+  isBackAllowed?: boolean
+}
+export const MobileNavHeader = (props: MobileNavHeaderProps) => {
+  const { isBackAllowed = true } = props
+  const history = useHistory()
+  const handleBackClick = () => {
+    history.goBack()
+  }
+  return (
+    <Flex
+      ph='xl'
+      pv='l'
+      w='100%'
+      css={(theme) => ({
+        backgroundColor: theme.color.background.white,
+        borderBottom: `solid 1px ${theme.color.border.default}`
+      })}
+      alignItems='center'
+      justifyContent='center'
+    >
+      <Flex
+        css={{ height: iconSizes.l, width: iconSizes.l }}
+        alignItems='center'
+      >
+        {isBackAllowed ? (
+          <PlainButton
+            css={{ padding: 0 }}
+            onClick={() => {
+              handleBackClick()
+            }}
+          >
+            <IconCaretLeft color='subdued' />
+          </PlainButton>
+        ) : null}
+      </Flex>
+      {/* mr to offset no icon on the right */}
+      <Box mr='xl' css={{ flex: '1' }}>
+        <IconAudiusLogoHorizontal
```

Comment:
not sure if you tried, but you can do `size='l'`

Reply:
Yeah I tried this initially, the issue is that the size prop shape things into a box, but for this we only want the height shaped to the preset size. I could be wrong but since these are all feeding through svgr + `Icon.tsx` I don't see a very obvious way to adjust the component logic so that the icon component automatically knows that `size='l'` only adjusts height. 
An alternative maybe is to have extra props for `sizeH` and `sizeW` or something 🤔

Reply:
(same as other comments, went ahead and added this to the batch ticket to tweak)
https://linear.app/audius/issue/C-3385/harmony-icon-components-tweaks